src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -357,6 +358,11 @@ private void initShuffleboard() {
 
 		angleSetpointEntry =
 				Shuffleboard.getTab("Launcher").add("Angle Setpoint", 0).withPosition(2, 2).getEntry();
+
+		Shuffleboard.getTab("Launcher")
+				.addDouble("setpoint + tol", () -> angleSetpoint - RPM_TOLERANCE);
+
+		Shuffleboard.getTab("Launcher").addBoolean("yay", this::isAtSpeed);
```

Comment:
If this is supposed to stay in the code, please give this a more descriptive name and specify a position (and size) so that it doesn't overlap with other entries.